paper.tex

```diff
@@ -69,6 +69,8 @@ \subsection{Detector models}
 
 \subsubsection{Image detector models}
 
+The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, pretrained image detection model. For finetuning the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```

Comment:
I would remove the first pretrained, as the model is not a foundation model and therefore I would not call it pretrained.
```suggestion
The \textbf{ai-image-det-resnet18} model is a community made, fine tuned, resnet18 based, image detection model. For fine-tuning the author kgmann on Hugging Face used the AI or Not dataset. ResNet-18 is a convolutional neural network that is 18 layers deep. It got pretrained by over a million images from the ImageNet database. \cite{https://arxiv.org/pdf/1512.03385.pdf}
```